service-registry/src/main/java/io/servicecomb/serviceregistry/client/ServiceRegistryClient.java

```diff
@@ -68,12 +75,24 @@
    */
   boolean registerSchema(String microserviceId, String schemaId, String schemaContent);
 
+  /**
+   * 
+   * 同步本地schema
+   */
+  boolean syncSchemasAll(Microservice microservice);
```

Comment:
we need a batch api to determine if some schemas exist.

Reply:
this time i use the batch upload interface that SC provided,if not good enough, we can use the better plan